packages/cli-core/package.json

```diff
@@ -18,6 +18,8 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@inquirer/prompts": "^3.0.0"
+    "@aws-amplify/platform-core": "^0.4.3",
```

Comment:
does this need `platform-core` or just `platform-types` ?

Reply:
Good catch! Neither of them is needed. c2e5194dd2